code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -1953,7 +1953,7 @@
 		if(target_facing != dir && target_facing != turn(dir,45) && target_facing != turn(dir,-45) ) //Have to be actually facing the target
 			continue
 		if(H.stat != DEAD && !(istype(H.buckled, /obj/structure/bed/nest) && H.status_flags & XENO_HOST) ) //No bully
-			var/extra_dam = rand(xeno_caste.melee_damage_lower, xeno_caste.melee_damage_upper) * (1 + round(rage * 0.01) ) //+1% bonus damage per point of Rage.relative to base melee damage.
+			var/extra_dam = rand(xeno_caste.melee_damage_lower, xeno_caste.melee_damage_upper) * round(0.25 + rage * 0.03, 0.01) //+25% +3% bonus damage per point of Rage.relative to base melee damage.
```

Comment:
move this value into either a config value, define or caste var and put the comments there.